[READY FOR REVIEW] Electronic Lab Notebooks Chapter

RichardJActon opened the pull request:
### Summary

Electronic lab notebooks are currently addressed in a [chapter in research data management on methods](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-methods). It is also mentioned in a [chapter in open research on open notebooks](https://the-turing-way.netlify.app/reproducible-research/open/open-notebooks). ELNs are an important enough subject to deserve their own chapter within research data management and this PR creates one. It inserts references to this new chapter in the two previous locations which mentioned ELNs. It also moves and slightly re-works the content on ELNs from previous section in methods, whilst providing an extended discussion on considerations when selecting and ELN solution.

Fixes #3434

[Direct link to the Chapter preview](https://deploy-preview-3466--the-turing-way.netlify.app/reproducible-research/rdm/rdm-elns#rr-rdm-elns)

### List of changes proposed in this PR (pull-request)

* Create new ELN chapter
* Link to it from the previous places that touch on ELNs
* Move and slightly re-work existing ELN content from methods to the new chapter

### What should a reviewer concentrate their feedback on?

- [ ] *lightly opinionated* as a free software guy lean quite heavily on the 'go with an open solution if you can' side of the scales, let me know if I've gone too far.
- [ ] Spelling a grammar, (I'm dyslexic so this can be pretty hit and miss)

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file: <!-- replace this text with the GitHub IDs of any new contributors -->

jezcope commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+* **Sustainability**: Choose a solution that has a larger chance of being around for a long time.
+* **Drawing**: Not all solutions have the tools or capabilities to include drawings or integration with drawing software.
+* **Security**: Before using an ELN solution you should check any backup plans and data security measures, especially if you're working with sensitive data.
+* **Laboratory situation**: Your lab may not have an internet collection or electronic tools could affect samples, reagents or magnets in the lab. 
```
Typo here: "collection" should be "connection"

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+  Or perhaps a foundation or other non-profit/academic organisation with robust funding.
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in it's native form by running the ELN application in a virutual machine or similar reproducible computational environment in the future should you need to.
```
```suggestion
Open solutions are more likely to provide access to data in it's native form.
```
It is not entirely clear to me what the second part of the sentence is conveying - maybe a rewrite?

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+Even if the tools are no longer maintained and in a state that can be used in production.
+They can still be used to read the data and interact with it in (mostly) the same way.
+The data will also likely be stored in an open format from which it can relatively easily be extracted and ported to a new format.
+This may still be a lot of work but it is much more attainable than reverse engeering a proprietary format for which there is no documentation or worse still dealing with encrypted files.
```
```suggestion
This may still be a lot of work but it is much more attainable than reverse engineering a proprietary format for which there is no documentation or worse - dealing with encrypted files.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+The data will also likely be stored in an open format from which it can relatively easily be extracted and ported to a new format.
+This may still be a lot of work but it is much more attainable than reverse engeering a proprietary format for which there is no documentation or worse still dealing with encrypted files.
+
+Most proprietary or open tools will permit you to export your data in one format or another, however the quality of this export is critical to scrutinise.
```
```suggestion
Most proprietary or open tools will permit you to export your data in one format or another.
The quality of this export is critical to scrutinise.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+
+Most proprietary or open tools will permit you to export your data in one format or another, however the quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it's the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
+Migration may cost you much of your data's structure.
```
```suggestion
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+The ability to save all your lab books as PDFs is fine but if it's the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
+Migration may cost you much of your data's structure.
+
+Proprietary solutions are incentivised to attempt to lock-in their customers so that the cost of switching to a competitor is high so be especially wary of poor export options.
```
```suggestion
Proprietary solutions are incentivised to attempt to lock-in their customers so that the cost of switching to a competitor is high - so be especially wary of poor export options.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+
+Proprietary solutions are incentivised to attempt to lock-in their customers so that the cost of switching to a competitor is high so be especially wary of poor export options.
+Note that this tends to change over time. 
+New providers are focused on user aquisition so data portability tends to be good at first but decline if a firm has to shift from a growth phase to a steady state that is sustainably profitable.
```
```suggestion
New providers are focused on user acquisition so data portability tends to be good at first but may decline if a firm adjusts their strategy.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption) making it impossible to migrate your local library to different reference managers without going through their online library feature.
+This occured after they had reached a high degree of market penetration and been aquired by Elsevier.
+
+Many open tools on the other hand have very robust exort and import options, as they are not subject to this incentives trap, though this is far from garunteed.
```
```suggestion
Many open tools on the other hand have very robust export and import options, as they are not subject to this incentives trap, though this is far from guaranteed.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+There remains the challenge when migrating of finding a tool that can import what your old tool exported, and retain most of all of its structure.
+This is where the development of standards for ELN formats could very valuable in the long run.
+
+This recent review {cite:ps}`higgins2022elns` provides a good overview of considerations when adopting an ELN solution. 
```
I feel it is better to start this section as a consideration of whether hosting a solution by a third party is an option, rather than starting with this resource. Instead I think it is better to put all the 'what ELN solution to pick' in one bullet point paragraph below.

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers as the hosting and support are no longer vertically integrated parts of the software as a service (SaaS) experience for you.
+
+There are many solutions to choose from: [How to pick an electronic laboratory notebook](https://doi.org/10.1038/d41586-018-05895-3) by {cite:ps}`Kwok2018eln` or [2019 Review of the Best Electronic Laboratory Notebooks](https://app.scientist.com/blog/2019/04/05/2019-review-of-the-best-electronic-laboratory-notebooks) may help you to find your solution!
```
Maybe a bullet point overview of resources that provide an overview or considerations when choosing an ELN?

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+Note that ELNs with APIs have the potential for integration with instruments that would permit direct deposition of data from connected instruments into ELNs or other connected research data management infrastructure.
+These sorts of integrations are especially useful in workflows at core facilities such as bioimaging facilities who can do things like automate a substancial part of the image metadata deposition into image data management tools such as [OMERO](https://www.openmicroscopy.org/omero/).
+
+ELNs offer many advantages over paper notebooks but they come with a considerable amount of additional complexity and a number of additional risks that it is important to have a plan to mitigate when proceeding with their adoption.
```
I would delete this paragraph or move it to the section below since it is not really about choosing a certain ELN solution.

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+(rr-rdm-open-elns)=
+## Select Open Infrastructure ELN Options
+
+They are each quite different but have many of the same core features.
```
```suggestion
Examples of Open ELN solutions are:
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+## Select Open Infrastructure ELN Options
+
+They are each quite different but have many of the same core features.
+For example rich text editing in a web browser.
```
```suggestion
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+
+They are each quite different but have many of the same core features.
+For example rich text editing in a web browser.
+The ability to upload files.
```
```suggestion
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+They are each quite different but have many of the same core features.
+For example rich text editing in a web browser.
+The ability to upload files.
+Sharing and permissions based on roles/groups.
```
```suggestion
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+Sharing and permissions based on roles/groups.
+
+*  [eLabFTW](https://www.elabftw.net/) 
+    *   Laboratory resource scheduling feature for booking things like hoods and microscopes, automatic mol file previews for molecules and proteins & support for free-hand drawing.
```
```suggestion
    *  Has features such as: Laboratory resource scheduling feature for booking things like hoods and microscopes, automatic mol file previews for molecules and proteins, and support for free-hand drawing.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+
+*  [eLabFTW](https://www.elabftw.net/) 
+    *   Laboratory resource scheduling feature for booking things like hoods and microscopes, automatic mol file previews for molecules and proteins & support for free-hand drawing.
+    *   The eLabFTW [site](https://www.elabftw.net/) and [documentation](https://doc.elabftw.net/) there is also a [demo](https://demo.elabftw.net/login.php) deployment that you can try out
```
```suggestion
    *   The eLabFTW [site](https://www.elabftw.net/) and [documentation](https://doc.elabftw.net/) has a [demo](https://demo.elabftw.net/login.php) deployment that you can try out.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+    *   The eLabFTW [site](https://www.elabftw.net/) and [documentation](https://doc.elabftw.net/) there is also a [demo](https://demo.elabftw.net/login.php) deployment that you can try out
+    *   Self-hosting is *relatively* simple [according to the documentation](https://doc.elabftw.net/install.html).
+    There is also a [paid support tier](https://www.deltablot.com/elabftw/) which would be recommend for any larger deployment to support the ongoing development of the project.
+    *   [Paid cloud hosting](https://www.deltablot.com/elabftw/) is available from the developer in a geographical region suited to your needs, a more expensive tier with hosting in France compliant with additional security and privacy certifications is available.
```
```suggestion
    *   [Paid cloud hosting](https://www.deltablot.com/elabftw/) is available from the developer in a geographical region suited to your needs. A more expensive tier with hosting in France compliant with additional security and privacy certifications is available.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+    There is also a [paid support tier](https://www.deltablot.com/elabftw/) which would be recommend for any larger deployment to support the ongoing development of the project.
+    *   [Paid cloud hosting](https://www.deltablot.com/elabftw/) is available from the developer in a geographical region suited to your needs, a more expensive tier with hosting in France compliant with additional security and privacy certifications is available.
+* [openBIS](https://openbis.ch/)
+    *   Good features for integrated metadata management for example linking to ontologies / controlled vocabularies.
```
```suggestion
    *   Includes features for integrated metadata management (for example linking to ontologies / controlled vocabularies).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+    *   [Paid cloud hosting](https://www.deltablot.com/elabftw/) is available from the developer in a geographical region suited to your needs, a more expensive tier with hosting in France compliant with additional security and privacy certifications is available.
+* [openBIS](https://openbis.ch/)
+    *   Good features for integrated metadata management for example linking to ontologies / controlled vocabularies.
+    This is based in a flexible object system for making similar entries.
```
```suggestion
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+* [openBIS](https://openbis.ch/)
+    *   Good features for integrated metadata management for example linking to ontologies / controlled vocabularies.
+    This is based in a flexible object system for making similar entries.
+    *   openBIS has an API and can integrate with jupyterhub for electronic lab notebooks.
```
```suggestion
    *   openBIS has an API and can integrate with Jupyterhub for Electronic Lab Notebooks.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+    *   Good features for integrated metadata management for example linking to ontologies / controlled vocabularies.
+    This is based in a flexible object system for making similar entries.
+    *   openBIS has an API and can integrate with jupyterhub for electronic lab notebooks.
+    *   Very feature rich LIMS system with optional integration of stores management with protocols and experiments including keeping track of bar-coded stocks.
```
```suggestion
    *   Has an inventory system with optional integration of stores management, as well as integration with protocols and experiments (including keeping track of bar-coded stocks).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+    *   Very feature rich LIMS system with optional integration of stores management with protocols and experiments including keeping track of bar-coded stocks.
+    *   You can get a feel for it in the [demo](https://openbis-eln-lims.ethz.ch/openbis/webapp/eln-lims/) deployment.
+    *   openBIS is a bit more complex to administer [based on its documentation](https://openbis.ch/index.php/docs/admin-documentation/).
+    It's a slightly older and more complex project than the others on this list, meaning it is very featureful and well tested against the needs of the groups at ETH Zurich.
```
```suggestion
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+    *   openBIS is a bit more complex to administer [based on its documentation](https://openbis.ch/index.php/docs/admin-documentation/).
+    It's a slightly older and more complex project than the others on this list, meaning it is very featureful and well tested against the needs of the groups at ETH Zurich.
+    *   Developed at ETH Zurich openBIS can be hosted for you under the openRDM service operated by ETH Zurich scientific IT services.
+    No fixed pricing is available cost would be dependent on your specific needs
```
```suggestion
    No fixed pricing is available - cost would be dependent on your specific needs.
    
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+    *   You can get a feel for it in the [demo](https://openbis-eln-lims.ethz.ch/openbis/webapp/eln-lims/) deployment.
+    *   openBIS is a bit more complex to administer [based on its documentation](https://openbis.ch/index.php/docs/admin-documentation/).
+    It's a slightly older and more complex project than the others on this list, meaning it is very featureful and well tested against the needs of the groups at ETH Zurich.
+    *   Developed at ETH Zurich openBIS can be hosted for you under the openRDM service operated by ETH Zurich scientific IT services.
```
```suggestion
    *   As openBIS is developed at ETH Zurich it can be hosted for you under the openRDM service operated by ETH Zurich scientific IT services.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+    *   OSF is oriented towards sharing and collaborating on your work, including the ability to generate DOIs and host pre-prints directly on the main instance.
+    *   It is free to use OSF at the main instance at [osf.io](https://osf.io/) so you can try it out there directly.
+    For larger data you must provide your own additional [storage addons, available from a number fo cloud storage providers](https://help.osf.io/article/395-storage-add-ons).
+    *   Whilst OSF can be hosted yourself this is presented by the project as for the purposes of development, and is not directly available as a paid service.
```
```suggestion
    *   OSF can be self-hosted but not as a paid service.
```

EstherPlomp reviewed (changes requested):
Looks very nice - thanks for adding! 

I have some structure concerns which I think are very easily addressed by adding another sub section on differences between paper and ELNs, instead of adding this information into the 'how to pick an ELN solution.

RichardJActon commented:
> Looks very nice - thanks for adding!
> 
> I have some structure concerns which I think are very easily addressed by adding another sub section on differences between paper and ELNs, instead of adding this information into the 'how to pick an ELN solution.

Thanks for the thoughtful review @EstherPlomp I agree about restructuring this a little - it does look like a bit of a wall of text at the moment when I actually look at it in the preview.

I actually want to avoid the 'paper vs ELN' then 'now I know I want an ELN which ELN should I pick' framing. I'm trying to frame the choice of ELN in terms of which ELNs best permit you to retain the advantages of paper AND benefit from the advantages of ELNs. I think that paper notebooks are actually THE most important point of comparison for ELN solutions NOT other ELNs, as paper is the default that they are trying to replace.

It does not matter how wiz-bang your ELNs features are if it can't do the fundamentals, there is of course an element of trade-off. However, there are certain things that paper comes with by default, that ELNs do not. These should probably be high priority features in most prospective ELN users' use cases, but are often not given enough attention - in my view.

I've got some thoughts on how to keep this framing and improve the structure that I'll play around with over the next little bit.

RichardJActon commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+* Secure storage and back up (with distinct failture modes from paper notebooks)
+* Tools for working with tabular data (calculations and formatting of tables and graphs)
+* Templates for documenting standard procedures
+* Laboratory inventories for managing and documenting samples, reagents and apparatus (They may have built in or integrate with independent laboratory inventory management systems (LIMS))
```
Addressed in [77a934b](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/77a934b71a56b1292ed5af62a6532a957cc5515f)

RichardJActon commented:
Thanks again @EstherPlomp if you get a chance I'd appreciate your input on my new structure I think it fixes the structural issue that you raised before.

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+Electronic data can *if manged properly* be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
+A hard drive or even a solid state drive however cannot be left in a draw for a decade or more and have a high likelihood of working without some amount of bitrot or compatability issue when plugged back in.
+
+Lab notebooks perform an archival function and proprietary formats are antithetical to this as they assume the institution which can act as a gatekeeper to the use of the proprietary format will outlive the need to archive the material.
```
```suggestion
Lab notebooks perform an archival function.
Proprietary formats are antithetical to this as this assumes that the proprietary format will remain functional for the archival period. 
```
Or something else to clarify this sentence

RichardJActon replied:
clarified this here [c567a05](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/c567a051c56e793020f50c6e73f8e7a450633286), &
[c7b38f0](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/c7b38f07b4f03e7c310da34e60c3b2f6ac167647) hopefully not too long now

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+
+Most proprietary or open tools will permit you to export your data in one format or another.
+The quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it's the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
```
```suggestion
The ability to save all your lab books as PDFs is fine but if it is the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+Most proprietary or open tools will permit you to export your data in one format or another.
+The quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it's the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
+The ability to export your data and retain it's structure is very important to evaluate.
```
```suggestion
The ability to export your data and retain the original structure is very important to evaluate.
```

RichardJActon replied:
Addressed in [e1318f5](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/e1318f5c6726d59c408b51fd05bf95b114e846b0)

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+
+Proprietary solutions are incentivised to attempt to lock-in their customers so that the cost of switching to a competitor is high - so be especially wary of poor export options.
+Note that this tends to change over time. 
+New providers are focused on user aquisition so data portability tends to be good at first but decline after a certain point in time.
```
```suggestion
New providers are focused on user acquisition so data portability tends to be good at first but may decline after a certain point in time.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+Proprietary solutions are incentivised to attempt to lock-in their customers so that the cost of switching to a competitor is high - so be especially wary of poor export options.
+Note that this tends to change over time. 
+New providers are focused on user aquisition so data portability tends to be good at first but decline after a certain point in time.
+Firms tends to follow a pattern of shifting from a user aquisiton phase with favourable terms, to a user retention phase with less favourable ones.
```
```suggestion
Firms tends to follow a pattern of shifting from a user acquisition phase with favourable terms, to a user retention phase with less favourable ones.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+Note that this tends to change over time. 
+New providers are focused on user aquisition so data portability tends to be good at first but decline after a certain point in time.
+Firms tends to follow a pattern of shifting from a user aquisiton phase with favourable terms, to a user retention phase with less favourable ones.
+Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption) making it impossible to migrate your local library to different reference managers without going through their online library feature.
```
```suggestion
Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption). 
This made it impossible to migrate your local library to different reference managers without going through the Mendeley online library feature.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+New providers are focused on user aquisition so data portability tends to be good at first but decline after a certain point in time.
+Firms tends to follow a pattern of shifting from a user aquisiton phase with favourable terms, to a user retention phase with less favourable ones.
+Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption) making it impossible to migrate your local library to different reference managers without going through their online library feature.
+This occured after they had reached a high degree of market penetration and been aquired by Elsevier.
```
```suggestion
This occurred after they had reached a high degree of market penetration and had been acquired by Elsevier.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in it's native form.
+For example keeping copies of the ELN software that can be run offline in a VM or similar computational environment in the future.
+This is typically much easier for open applications and can be impossible for proprietary ones.
+Even if the tools are no longer maintained and in a state that can be used in production they can still be used to read the data and interact with it in (mostly) the same way.
```
```suggestion
Even if the tools are no longer maintained they can still be used to read the data and interact with it in (mostly) the same way.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+This is typically much easier for open applications and can be impossible for proprietary ones.
+Even if the tools are no longer maintained and in a state that can be used in production they can still be used to read the data and interact with it in (mostly) the same way.
+The data will also likely be stored in an open format from which it can relatively easily be extracted and ported to a new format.
+This may still be a lot of work but it is much more attainable than reverse engineering a proprietary format for which there is no documentation or worse - dealing with encrypted files.
```
```suggestion
This may still be a lot of work but it is much more attainable than reverse engineering a proprietary format for which there is no documentation, or worse - dealing with encrypted files.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+
+You can pay for third party hosting and administration of open source applications.
+This is a solution when you don't have the expertise or internal resources to administer a self-hosted instance of an open-source ELN solution. 
+In this case you get the best of both worlds, the benefits of professional support and the reassurance of an open solution.
```
```suggestion
In this case you get the best of both worlds: the benefits of professional support and the reassurance of an open solution.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+In this case you get the best of both worlds, the benefits of professional support and the reassurance of an open solution.
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers.
+Conventionl software as a service (SaaS) platforms are highly verically integrated, the software development, hosting, and administration are package deal that you must take or leave as is.
```
```suggestion
Conventional Software as a Service (SaaS) platforms are highly vertically integrated: the software development, hosting, and administration are package deal that you must take or leave as is.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers.
+Conventionl software as a service (SaaS) platforms are highly verically integrated, the software development, hosting, and administration are package deal that you must take or leave as is.
+Using open tools breaks this up affording you the flexibility to change any of these if the need arises and you have the resources to do so, it puts you in a better bargining position.
```
```suggestion
Using open tools breaks this up, affording you the flexibility to change any of these if the need arises.
```
Or a different edit to make this sentence a bit more readable

RichardJActon replied:
Addressed in: [bfbcaec](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/bfbcaec15b27994c94556acb918eaa143a4dcf28)

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+### Automation and integration
+
+ELNs with APIs have the potential for integration with instruments that would permit direct deposition of data from connected instruments into ELNs or other connected research data management infrastructure.
+These sorts of integrations are especially useful in workflows at core facilities such as bioimaging facilities who can do things like automate a substancial part of the image metadata deposition into image data management tools such as [OMERO](https://www.openmicroscopy.org/omero/).
```
```suggestion
These sorts of integrations are especially useful in workflows at core facilities. 
For example, bioimaging facilities can automate a substantial part of the image metadata deposition into image data management tools such as [OMERO](https://www.openmicroscopy.org/omero/).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+(rr-rdm-elns-standards)=
+## ELN standards and Project governance
+
+ELNs offer many advantages over paper notebooks but they come with a considerable amount of additional complexity and a number of additional risks that it is important to have a plan to mitigate when proceeding with their adoption.
```
```suggestion
ELNs offer many advantages over paper notebooks.
Nevertheless, ELNs come with a considerable amount of additional complexity and additional risks - so it is important to have a mitigation plan.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+
+ELNs offer many advantages over paper notebooks but they come with a considerable amount of additional complexity and a number of additional risks that it is important to have a plan to mitigate when proceeding with their adoption.
+The ELN space is still quite young and rapildy changing.
+ELNs represent a key component of digital infrastructure for the future of research.
```
```suggestion
```

RichardJActon replied:
Did this in: [327fba9](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/327fba994f8b0e9f0d44d30deeaf90b7858478eb)

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+ELNs offer many advantages over paper notebooks but they come with a considerable amount of additional complexity and a number of additional risks that it is important to have a plan to mitigate when proceeding with their adoption.
+The ELN space is still quite young and rapildy changing.
+ELNs represent a key component of digital infrastructure for the future of research.
+As such it is important to think about how the academic commuity collectively wants this infrastructure to be owned and governed in the long run at this stage in the adoption of ELNs.
```
```suggestion
As such it is important to think about how the academic community wants the ELN infrastructure to be owned and governed in the long run.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+The ELN space is still quite young and rapildy changing.
+ELNs represent a key component of digital infrastructure for the future of research.
+As such it is important to think about how the academic commuity collectively wants this infrastructure to be owned and governed in the long run at this stage in the adoption of ELNs.
+An interesting model to consider emulating here is [moodle](https://moodle.com/) an open source electronic learning environment popular with universities.
```
```suggestion
An interesting model to consider emulating here is [moodle](https://moodle.com/), an open source electronic learning environment popular with universities.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+ELNs represent a key component of digital infrastructure for the future of research.
+As such it is important to think about how the academic commuity collectively wants this infrastructure to be owned and governed in the long run at this stage in the adoption of ELNs.
+An interesting model to consider emulating here is [moodle](https://moodle.com/) an open source electronic learning environment popular with universities.
+It offers commercial support and hosting and has a network of developers across the education sector who contribute to the core project as well as creating custom integrations and extensions to meet their particular needs.
```
```suggestion
It offers commercial support and hosting and has a network of developers across the education sector who contribute to the core project, as well as creating custom integrations and extensions to meet their particular needs.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+This structure allows institutions to pool resources to fund the development of features that they all need and keeps this infrastructural code in common ownership, avoiding the worst lock-in issues arising from the use of proprietary solutions.
+This more open distributed model also allows individuals to work on their own custom extensions and integrations instead of being reliant on the ability and willingness of a company to implement a feature that you need.
+
+Many open tools have very robust export and import options, as they are not subject to the incentives trap faced by proprietary offerings, though this is far from guaranteed.
```
It looks like this paragraph fits in better with the lock in section above?

RichardJActon replied:
Addressed in: [1318565](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/1318565ac1917a33d1191552f368374fef03242a)

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+## Resources for choosing ELNs
+
+- [How to pick an electronic laboratory notebook](https://doi.org/10.1038/d41586-018-05895-3) by {cite:ps}`Kwok2018eln`
+- [2019 Review of the Best Electronic Laboratory Notebooks](https://app.scientist.com/blog/2019/04/05/2019-review-of-the-best-electronic-laboratory-notebooks) may help you to find your solution!
```
```suggestion
- [2019 Review of the Best Electronic Laboratory Notebooks](https://app.scientist.com/blog/2019/04/05/2019-review-of-the-best-electronic-laboratory-notebooks).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+
+- [How to pick an electronic laboratory notebook](https://doi.org/10.1038/d41586-018-05895-3) by {cite:ps}`Kwok2018eln`
+- [2019 Review of the Best Electronic Laboratory Notebooks](https://app.scientist.com/blog/2019/04/05/2019-review-of-the-best-electronic-laboratory-notebooks) may help you to find your solution!
+- This [guide on choosing an ELN from Simon Bungers of Labfolder](https://labfolder.com/electronic-lab-notebook-eln-research-guide/) is also worth a read.
```
```suggestion
- [Guide on choosing an ELN from Simon Bungers of Labfolder](https://labfolder.com/electronic-lab-notebook-eln-research-guide/).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+(rr-rdm-eln-open-elns)=
+## Select Open Infrastructure ELN Options
+
+These are each quite different but have many of the same core features.
```
```suggestion
The following ELN options are each quite different but have many of the same core features.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+        A more expensive tier with hosting in France compliant with additional security and privacy certifications is available.
+
+* [openBIS](https://openbis.ch/)
+    *   Robust features for integrated metadata management for example linking to ontologies / controlled vocabularies.
```
```suggestion
    *   Robust features for integrated metadata management, for example, linking to ontologies / controlled vocabularies.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+* [openBIS](https://openbis.ch/)
+    *   Robust features for integrated metadata management for example linking to ontologies / controlled vocabularies.
+    *   openBIS has an API and can integrate with Jupyterhub for Electronic Lab Notebooks.
+    *   Complete laboratory inventory managment system (LIMS) including stores management integrated with protocols and experiment records, including keeping track of bar-coded stocks.
```
```suggestion
    *   Complete Laboratory Inventory Nanagement System (LIMS) including stores management integrated with protocols and experiment records, including keeping track of bar-coded stocks.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+    
+* [OSF](https://osf.io/)
+    *   OSF is oriented towards sharing and collaborating on your work, including the ability to generate DOIs and host pre-prints directly on the main instance.
+    *   It is free to use OSF at the main instance at [osf.io](https://osf.io/) so you can try it out there directly.
```
```suggestion
    *   It is free to use OSF at the main instance at [osf.io](https://osf.io/).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+* [OSF](https://osf.io/)
+    *   OSF is oriented towards sharing and collaborating on your work, including the ability to generate DOIs and host pre-prints directly on the main instance.
+    *   It is free to use OSF at the main instance at [osf.io](https://osf.io/) so you can try it out there directly.
+        For larger data you must provide your own additional [storage addons, available from a number fo cloud storage providers](https://help.osf.io/article/395-storage-add-ons).
```
```suggestion
        For larger data you must provide your own additional [storage add-ons, available from a number of cloud storage providers](https://help.osf.io/article/395-storage-add-ons).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+    *   OSF is oriented towards sharing and collaborating on your work, including the ability to generate DOIs and host pre-prints directly on the main instance.
+    *   It is free to use OSF at the main instance at [osf.io](https://osf.io/) so you can try it out there directly.
+        For larger data you must provide your own additional [storage addons, available from a number fo cloud storage providers](https://help.osf.io/article/395-storage-add-ons).
+    *   Whilst you could deploy your own self-hosted OSF this is not how it is intended to be used except for development.
```
```suggestion
    *   Whilst you could deploy your own self-hosted OSF this is not how it is intended to be used (except for development).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+    *   Whilst you could deploy your own self-hosted OSF this is not how it is intended to be used except for development.
+    *   The OSF has strong sharing features, making it easy to share parts of your ELN publicly.
+
+When using an open solution it is always important to consider how you can contribute to its ongoing development and maintenence, for example by donating to the project.
```
```suggestion
When using an open solution it is always important to consider how you can contribute to its ongoing development and maintenance, for example by donating to the project.
```

EstherPlomp reviewed (commented):
Thanks for incorporating the feedback @RichardJActon! I think the structural concerns are now addressed - thank you! 

I now just have some consistency and spelling suggestions - looks like we're almost there :)

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,238 @@
+
+* A text editor with similar functions as a paper notebook
+* A search function
+* Secure storage and back up (with distinct failture modes from paper notebooks)
```
```suggestion
* Secure storage and back up (with distinct failure modes from paper notebooks)
```

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,238 @@
+Incomplete or inflexible ELN solutions which increase friction in workflows can cause issues.
+People will avoid using ill-fitting tools that they perceive them to be getting in their way, especially scientists.
+They are creative, stubborn, resourceful, and impatiently focused on anwsering their research questions with watever tools are available to them. 
+Finding software with the flexibility, capability, and scope to keep up is not easy and if it doesn't researchers won't use it - at least not as you indented.
```
```suggestion
Finding software with the flexibility, capability, and scope to keep up is not easy and if it doesn't researchers won't use it - at least not as you intended.
```

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,238 @@
+    -   If there are paper and potentially multiple electronic copies of some information - what happens when it does not agree? What is the source of truth?
+-   Partial adoptions can lead to information which was previous predictably structured in lab notebooks being spread across multiple IT systems and physical copies
+-   Practical use and interface issues
+    -   Laptops or tablets can intrduce interface barriers 
```
```suggestion
    -   Laptops or tablets can introduce interface barriers 
```

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,173 @@
+Electronic data can *if manged properly* be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
+A hard drive or even a solid state drive however cannot be left in a draw for a decade or more and have a high likelihood of working without some amount of bitrot or compatability issue when plugged back in.
+
+Lab notebooks perform an archival function and proprietary formats are antithetical to this as they assume the institution which can act as a gatekeeper to the use of the proprietary format will outlive the need to archive the material.
```
I like @EstherPlomp's suggestion here. The intended meaning of the previous version is unclear to me.

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,238 @@
+
+Lab notebooks perform an archival function and proprietary formats are antithetical to this as they assume the institution which can act as a gatekeeper to the use of the proprietary format will outlive the need to archive the material.
+When choosing an archival format one seeks to maximize the likelihood that one can recover the relevant information from that format.
+Using a proprietary solution for archival purposes is talking a needless risk with the future of your data.
```
```suggestion
Using a proprietary solution for archival purposes is taking a needless risk with the future of your data.
```

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,238 @@
+
+Most proprietary or open tools will permit you to export your data in one format or another.
+The quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it is the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
```
```suggestion
The ability to save all your lab books as PDFs is fine but if it is the only option you may lose a lot of metadata, and the ability to import your data into a new solution. 
```

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,238 @@
+  Or perhaps a foundation or other non-profit/academic organisation with robust funding.
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in it's native form.
```
```suggestion
Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in its native form.
```

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,238 @@
+The choice of ELN is an incredibly important decision, and one that your organisation will likely have live with for years, or even decades.
+You are putting the record of your research into the hands of the tool that you choose, and entering into a long-term relationship with the provider of your ELN solution.
+You are also making a choice about a tool that many of the people in your organisation will interface with closely every day.
+Having to use a tool that is not well fit to it function on a daily basis can be a major problem for productivity and morale.
```
```suggestion
Having to use a tool that is not well fit to its function on a daily basis can be a major problem for productivity and morale.
```

da5nsy reviewed (approved):
Read through the content, reviewing content, and also spelling/grammar. All LGTM!

Thanks for writing this important section! Congratulations! 🎉

da5nsy commented:
> lightly opinionated as a free software guy lean quite heavily on the 'go with an open solution if you can' side of the scales, let me know if I've gone too far.

I'm probably of a similar nature, so I'm not sure if I'm the best to comment on this (!), but _I_ think it's great as is! haha

da5nsy commented:
I'm not sure if it would be relevant/useful to add anything about this to this PR, and so I'm mainly sharing because you might find it interesting @RichardJActon, but I recently set up a collaborative lab notebook system for the lab I was doing my postdoc at, using some open source software called Logseq. I wrote about it in a blog post [here](https://www.dannygarside.co.uk/blog/An-open-source-knowledge-management-system-for-the-lab/) if you're curious to read more. This is definitely at the far end of the "minimal but flexible" spectrum - it can handle text/pictures/links etc but it doesn't have any of the bells and whistles like inventory management.

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+
+When picking an ELN remember to always compare the ELN to a paper notebook not just to other ELNs, it's likely paper that you are replacing so it is the benchmark against which to compare.
+Consider the differences between a paper copy of a lab notebook and an electronic copy.
+Consider also which are the properties of a paper copy that it is important that you are able to retain when adopting an electronic alternative.
```
@EstherPlomp I think your version is missing a word somewhere

RichardJActon replied:
addressed in [d9c1d16](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/d9c1d167af40ed9ee0b5b826b904c5e5919f4281)

RichardJActon commented:
> I'm not sure if it would be relevant/useful to add anything about this to this PR, and so I'm mainly sharing because you might find it interesting @RichardJActon, but I recently set up a collaborative lab notebook system for the lab I was doing my postdoc at, using some open source software called Logseq. I wrote about it in a blog post [here](https://www.dannygarside.co.uk/blog/An-open-source-knowledge-management-system-for-the-lab/) if you're curious to read more. This is definitely at the far end of the "minimal but flexible" spectrum - it can handle text/pictures/links etc but it doesn't have any of the bells and whistles like inventory management.

Thanks for the feedback @da5nsy. Logseq is the note taking software that I use daily for my general notes and I had though that is might adapt OK to being a lab notebook once it matures a little, I'd be interested to hear how it goes for you once your group has spent some more time with it. The 'everything in plain text' aspect of it is really appealing for a strong guarantee of longevity and portability, and for me management with git - though that's double edged if your working with people who don't use git. I had contemplated a section on 'personal knowledge management' tools as there is some overlap with ELNs but that's a whole extra section as there are so many of these I've got a list of ~18 and that's just the open source or partially open ones.

(FYI hoping to get everything outstanding on this PR resolved before next week)

da5nsy commented:
> I'd be interested to hear how it goes for you once your group has spent some more time with it

We've been using it for about a year now (slowly ramping up - it's only been a couple of months since everyone was using it on a daily basis, before that it was mainly just me haha) and it seems to be going really well! The main challenges were social rather than technical - convincing everyone to use the same formatting, but honestly - that's only to aid hypothetical future searches, so it's a "nice to have".

> The 'everything in plain text' aspect of it is really appealing for a strong guarantee of longevity and portability

That was the real biggy for me.

> and for me management with git - though that's double edged if your working with people who don't use git

I see it as a training opportunity haha! But yes, seriously, your point is valid.

> I had contemplated a section on 'personal knowledge management' tools as there is some overlap with ELNs but that's a whole extra section as there are so many of these I've got a list of ~18 and that's just the open source or partially open ones.

Yeah it's a can of worms. I do wonder if we might be able to sneak in a sentence somewhere along the lines of "It is also possible to use general purpose note-taking software as a lab notebook. This can offer the advantages of simplicity, flexibility, and a greater range of software choice compared to specific ELN software, but comes with the cost of not being as feature-rich." What do you think?

> (FYI hoping to get everything outstanding on this PR resolved before next week)

🎉 
This topic is a special interest of mine, so do ping me if I can assist in any way.

RichardJActon commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+    -   People dump a copy of their work into the ELN as only often as necesary for compliance.
+-   Consistency issues
+    -   If there are paper and potentially multiple electronic copies of some information what happens when it does not agree? - what's the source of truth?
+-   Partial adoptions can lead to information which was previous predictably structured in lab notebooks being spread across multiple IT systems and physical copies
```
addressed in [379d592](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/379d59258b102d694f3abdae53a3964993ad349a)

RichardJActon commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+-   Practical use and interface issues
+    -   Laptops or tablets can intrduce interface barriers 
+        -   Something as simple as the use of touchscreens and trackpads with gloves
+        -   Free-hand drawing or use of notation not easily represented in simple text such as mathematical and chemical notation
```
addressed in [cd2c294](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/cd2c29402bd13b20ceaaf6d1023a2b72614fd9df)

RichardJActon commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+    -   Laptops or tablets can intrduce interface barriers 
+        -   Something as simple as the use of touchscreens and trackpads with gloves
+        -   Free-hand drawing or use of notation not easily represented in simple text such as mathematical and chemical notation
+        -   Performant native applications for devices are not always available, notebooks only accessible through web apps can have performance issues that make their interfaces frustrating to use
```
addressed in [a630127](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/a630127c382766890264eb205403e3f84abbbf97)

RichardJActon commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+        -   In a physical book people use a lot they remember spacially where things are they pick it up and open it immediately to roughly the right page
+
+-   lack of sufficiently available training on more complex aspects of the ELN solution leading to poor, mis-use, or under-use of ELN features
+    -   Extra expense, maintenence & procurement issues with hardware suitable for use in your research environment
```
addressed in [21b037c](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/21b037cce5905da84386ad5e31826c758a128e95)

RichardJActon commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,234 @@
+Few active measures are needed to maintain the data in your paper notebooks, they are vulnerable as they exist in only one copy. 
+Nevertheless, as long as they are kept in a cool, dry and dark spot for example a fireproof safe they will likely last decades.
+Electronic data requires much more active upkeep.
+Electronic data can *if manged properly* be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
```
Addressed in [aabc7fc](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/aabc7fc52ce5407af38fdc739cf16b9418968658)

RichardJActon commented:
Thanks again @EstherPlomp & @da5nsy for the feedback on this I think that is is much improved as a result. If there's anything else about how I've addressed the points you've raised you'd like me to tweak let me know, otherwise I'm hoping this is ready to merge now.

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Check if your institution is offering a solution that you can use without additional costs.
+* **File format**: Always check if files can be exported in your preferred file formats to avoid format lock-in.
+Some ELNs also have API's that allow integration with other software and workflows. 
+* **Vendor-lock-in**: Once you're using a certain solution you may become dependent on it.
```
Maybe a reference to the lock-in section below?

RichardJActon replied:
Addressed in [7322151](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/7322151f466607fe81a458c5ef56e89bc3f16dea)

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+        -   Free-hand drawing is not always well supported which can be necessary when using notation that is not easily represented in simple text such as mathematical and chemical notation
+        -   Client issues, such as native mobile applications with incomplete featuresets, slow web-apps that may require an always-on connection and not be able to cache local changes
+        -   No spatial memory, whereas in a physical book people may remember where they wrote things and can open it roughly on the right page
+        -   Extra expense, maintenence & procurement issues with hardware suitable for use in your research environment
```
```suggestion
        -   Extra expense, maintenance and procurement issues with hardware suitable for use in your research environment
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+### Always compare to paper notebooks
+
+When picking an ELN remember to always compare the ELN to a paper notebook and not just to other ELNs.
+It is likely paper that you are replacing so this is the benchmark against which to compare.
```
```suggestion

```
I think this is clear from the first sentence.

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+
+When picking an ELN remember to always compare the ELN to a paper notebook and not just to other ELNs.
+It is likely paper that you are replacing so this is the benchmark against which to compare.
+Consider the differences between a paper copy of a lab notebook and an electronic copy.
```
```suggestion

```
I think this is clear from the first sentence and the sentence after this one.

RichardJActon replied:
I appreciate that these points may seem redundant but they are all making a slightly different points.

- when you've got your matrix of ELN solutions don't forget to have paper in there too
- why have paper in there: because that's the thing you are replacing so it's the most relevant point of comparison
- don't forget to add the properties of paper notebooks that might be missing from the all the ELN feature lists you've encountered to your comparison matrix
- If one of those features of paper is a deal breaker and is not present in any of the ELNs then you may need to consider other options or delay ELN adoption until you can find one that does have this feature of paper

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+
+Incomplete or inflexible ELN solutions which increase friction in workflows can cause issues.
+People will avoid using ill-fitting tools that they perceive them to be getting in their way, especially scientists.
+They are creative, stubborn, resourceful, and impatiently focused on anwsering their research questions with watever tools are available to them. 
```
```suggestion
They are creative, stubborn, resourceful, and impatiently focused on answering their research questions with whatever tools are available to them. 
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Finding software with the flexibility, capability, and scope to keep up is not easy and if it doesn't researchers won't use it - at least not as you intended.
+
+-   Shadow IT/notebooks
+    -   People using their own unsactioned solutions which may be a compliance issue and introduce risks that you have not accounted for
```
```suggestion
    -   People using their own unsanctioned solutions which may be a compliance issue and introduce risks that you have not accounted for
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+When picking an ELN remember to always compare the ELN to a paper notebook and not just to other ELNs.
+It is likely paper that you are replacing so this is the benchmark against which to compare.
+Consider the differences between a paper copy of a lab notebook and an electronic copy.
+Consider also which properties of a paper copy is it important to you that you are able to retain when adopting an electronic alternative.
```
```suggestion
Consider which properties of a paper copy are important to you and which should be retained when adopting an electronic alternative.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Proprietary formats generally have a single implementation from their creator, this creates a single point of failure.
+Open formats can have multiple different implementations of tools which read them.
+If there is a good enough specification then new tools can even be written for the format if none of the originals are available anymore.
+Proprietary tools way require some kind of license activation process to use this may not be possible if their vendor no longer exists.
```
```suggestion
Proprietary tools may require some kind of license activation process to use this may not be possible if their vendor no longer exists.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Electronic data can, *if managed properly*, be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
+A hard drive, or even a solid state drive, however, cannot be left in a draw for a decade or more and have a high likelihood of working without some amount of bit-rot or compatibility issues when plugged back in.
+
+Lab notebooks perform an archival function.
```
```suggestion
For digital data it is important for archiving and maintaining the data to use formats that maximize the likelihood of recovering relevant information.
```
This paragraph starts as something which should probably be the first paragraph of this section, but then only talks about digital/proprietary formats, so I would lean into that second bit and rewrite the first lines.

RichardJActon replied:
Addressed in [a63fdec](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/a63fdec94d17943207bf2f62169d45151de10099)

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+If there is a good enough specification then new tools can even be written for the format if none of the originals are available anymore.
+Proprietary tools way require some kind of license activation process to use this may not be possible if their vendor no longer exists.
+Using a proprietary solution for archival purposes is therefore taking a needless risk with the future of your data.
+Your data's fate is tied to the format in which it is stored choose it wisely.
```
```suggestion
Your data's fate is tied to the format in which it is stored so choose it wisely.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Most proprietary or open tools will permit you to export your data in one format or another.
+The quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it is the only option you may lose a lot of metadata, and the ability to import your data into a new solution. 
+The ability to export your data and retain the original structure is very important to evaluate.
```
```suggestion
The ability to export your data and retain the original structure is very important.
```

RichardJActon replied:
This is sometimes a matter of degree, you may be able to retain most of the information and the losses are acceptable to you when performing a migration. So the degree to which it is possible to 'losslessly' export your data is the criterion that I'm suggesting people consider when evaluating an ELN.

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in its native form.
+For example keeping copies of the ELN software that can be run offline in a VM or similar computational environment in the future.
```
```suggestion
For example keeping copies of the ELN software that can be run offline in a virtual machine (VM) or similar computational environment in the future.
```
Or add it to the glossary as a definition? The abbreviation hasn't been used before so should be explained in some way

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers.
+Conventional Software as a Service (SaaS) platforms are highly vertically integrated: the software development, hosting, administration, and support are package deal that you must take or leave as is.
+When using open tools each of these layers can seperated, though they do not need to be.
```
```suggestion
When using open tools each of these layers can separated, though they do not need to be.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+This means that you retain the option to change providers.
+Conventional Software as a Service (SaaS) platforms are highly vertically integrated: the software development, hosting, administration, and support are package deal that you must take or leave as is.
+When using open tools each of these layers can seperated, though they do not need to be.
+This gives you the flexibility to change your provider of any of them indepentenly.
```
```suggestion
This gives you the flexibility to change your provider of any of them independently.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Software which most institutions will already have, meaning they will not incurr additional costs if they use OneNote.
+
+Other generic note taking applications have featuresets which are not dissimilar to ELN solutions.
+For example open source applications like [logseq](https://logseq.com/) or [zettlr](https://www.zettlr.com/), and proprietary ones like: [obsidian](https://obsidian.md/) or [notion](https://www.notion.so/).
```
```suggestion
For example, open source applications like [logseq](https://logseq.com/) or [zettlr](https://www.zettlr.com/), and proprietary ones like: [obsidian](https://obsidian.md/) or [notion](https://www.notion.so/).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+For example, bioimaging facilities can automate a substantial part of the image metadata deposition into image data management tools such as [OMERO](https://www.openmicroscopy.org/omero/).
+
+(rr-rdm-elns-standards)=
+## ELN standards and Project governance
```
```suggestion
## ELN standards and project governance
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+This structure allows institutions to pool resources to fund the development of features that they all need and keeps this infrastructural code in common ownership, avoiding the worst lock-in issues arising from the use of proprietary solutions.
+This more open distributed model also allows individuals to work on their own custom extensions and integrations instead of being reliant on the ability and willingness of a company to implement a feature that you need.
+
+In the absense of an open standard for representing electronic lab notebook content using an open ELN solution is still no guarantee of data portability and interoperatbility.
```
```suggestion
In the absence of an open standard for representing electronic lab notebook content using an open ELN solution is still no guarantee of data portability and interoperability.
```

I'm not sure what this sentence means: Even if there is a standard it doesn't mean that data is portable/interoperable? Or even if there is no standard this could still be the case?

RichardJActon replied:
Addressed in [f64204a](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/f64204a51b90b4aad4ed831dfd8af15a5d56f531)

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+This more open distributed model also allows individuals to work on their own custom extensions and integrations instead of being reliant on the ability and willingness of a company to implement a feature that you need.
+
+In the absense of an open standard for representing electronic lab notebook content using an open ELN solution is still no guarantee of data portability and interoperatbility.
+Such a standard would, if properly adhered too, could reduce the lock-in risk associated with using using both open and proprietary tools.
```
```suggestion
Such a standard would, if properly adhered too, reduce the lock-in risk associated with using both open and proprietary tools.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Some tools are better suited to be used 'off-label' as ELNs than others.
+Some may have significant drawbacks compared to purpose built solutions, despite being better at some things than currently available dedicated ELN options.
+
+Microsoft OneNote is quite popular as an ELN despite a proprietary notebook format with limited export options and that [microsoft has entertained getting rid of the good (desktop) version this of application in favour of its much less featureful 'web' version in the past. Though they have apparently relented, for now.](https://www.youtube.com/watch?v=4IkUfh05TGE).
```
```suggestion
Microsoft OneNote is quite popular as an ELN despite a proprietary notebook format with limited export options and that [Microsoft has entertained getting rid of the good (desktop) version this of application in favour of its much less featureful 'web' version in the past. Though they have apparently relented, for now](https://www.youtube.com/watch?v=4IkUfh05TGE).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+When using an open solution it is always important to consider how you can contribute to its ongoing development and maintenance, for example by donating to the project.
+If it makes sense given the makeup of your organisation and that of the project you may also be able to contribute developer time to the project.
+
+(rr-rdm-using-non-elns-for-eln-things)
```
```suggestion
(rr-rdm-using-non-elns-for-eln-things)=
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+OneNote is a good experience as a free-form note taking tool especially for hand-written or drawn notes and automatic linking of anotations to audio timestamps.
+The lack of a good version history is a distinct downside.
+Its popularity likely stems, in part, from its availability in the Microsoft office suite.
+Software which most institutions will already have, meaning they will not incurr additional costs if they use OneNote.
```
```suggestion
```
I would combine these two sentences, otherwise it is not very clear what you mean by the current one.

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Microsoft OneNote is quite popular as an ELN despite a proprietary notebook format with limited export options and that [microsoft has entertained getting rid of the good (desktop) version this of application in favour of its much less featureful 'web' version in the past. Though they have apparently relented, for now.](https://www.youtube.com/watch?v=4IkUfh05TGE).
+OneNote is a good experience as a free-form note taking tool especially for hand-written or drawn notes and automatic linking of anotations to audio timestamps.
+The lack of a good version history is a distinct downside.
+Its popularity likely stems, in part, from its availability in the Microsoft office suite.
```
```suggestion
Its popularity likely stems, in part, from its availability in the Microsoft Office suite (software already provided by institutions which do not incur additional costs).
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+Some may have significant drawbacks compared to purpose built solutions, despite being better at some things than currently available dedicated ELN options.
+
+Microsoft OneNote is quite popular as an ELN despite a proprietary notebook format with limited export options and that [microsoft has entertained getting rid of the good (desktop) version this of application in favour of its much less featureful 'web' version in the past. Though they have apparently relented, for now.](https://www.youtube.com/watch?v=4IkUfh05TGE).
+OneNote is a good experience as a free-form note taking tool especially for hand-written or drawn notes and automatic linking of anotations to audio timestamps.
```
```suggestion
OneNote is a good experience as a free-form note taking tool especially for hand-written or drawn notes and automatic linking of annotations to audio timestamps.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+(rr-rdm-using-non-elns-for-eln-things)
+## Using non-ELNs for ELN things
+
+Note taking and personal knowledge management (PKM) applications of various kinds have been pushed into service as ELNs by many individuals and institutions, despite not being designed specifically for this tasks.
```
```suggestion
Note taking and personal knowledge management (PKM) applications of various kinds have been pushed into service as ELNs by many individuals and institutions, despite not being designed specifically for this task.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+* [openBIS](https://openbis.ch/)
+    *   Robust features for integrated metadata management, for example, linking to ontologies / controlled vocabularies.
+    *   openBIS has an API and can integrate with Jupyterhub for Electronic Lab Notebooks.
+    *   Complete Laboratory Inventory Nanagement System (LIMS) including stores management integrated with protocols and experiment records, including keeping track of bar-coded stocks.
```
```suggestion
    *   Complete LIMS where storage is integrated with protocols and experiment records, including keeping track of bar-coded stocks.
```

EstherPlomp commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,261 @@
+        No fixed pricing is available - cost would be dependent on your specific needs.
+    
+* [OSF](https://osf.io/)
+    *   OSF is oriented towards sharing and collaborating on your work, including the ability to generate DOIs and host pre-prints directly on the main instance.
```
```suggestion
    *   OSF is oriented towards sharing and collaborating on your work, including the ability to generate DOIs and host pre-prints directly on the main instance.
    See {ref}`OSF<rr-rdm-repository-osf>` for more information.
```

EstherPlomp reviewed (commented):
Looks great - thanks for updating! 

I only have some spelling corrections and minor suggestions now. I'm happy to approve once we get the spelling sorted! 

(Thanks for the wait - was on holidays!)

EstherPlomp commented:
Perhaps another add for the additional resources! An ELN finder that I found out about today: https://eln-finder.ulb.tu-darmstadt.de/home

EstherPlomp commented:
It looks like you added all the changes/suggestions? Shall I review again or are you still working on something? No rush - please let me know whenever it is ready to review so we can get this merged!

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,262 @@
+(rr-rdm-elns-choosing-importance)=
+### Importance of getting it right
+
+Many organisations offer software that purports to solve the problem of ELNS.
```
```suggestion
Many organisations offer software that purports to solve the problem of ELNs.
```

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,262 @@
+### Failure modes in user adoption of ELNs
+
+Incomplete or inflexible ELN solutions which increase friction in workflows can cause issues.
+People will avoid using ill-fitting tools that they perceive them to be getting in their way, especially scientists.
```
```suggestion
People will avoid using ill-fitting tools that they perceive to be getting in their way, especially scientists.
```

da5nsy commented on book/website/reproducible-research/rdm/rdm-elns.md:
```diff
@@ -0,0 +1,262 @@
+-   Consistency issues
+    -   If there are paper and potentially multiple electronic copies of some information - what happens when it does not agree? What is the source of truth?
+- Sprawl
+    -   Partial adoptions can lead to different pieces of information being stored in different systems which might not be connected making it harder to record and find information which was previous in one place.
```
```suggestion
    -   Partial adoptions can lead to different pieces of information being stored in different systems which might not be connected making it harder to record and find information which was previously in one place.
```

EstherPlomp reviewed (approved):
Looks good to me! Thanks for doing all the revisions!
